modules/apps/headless/headless-delivery/headless-delivery-impl/src/main/java/com/liferay/headless/delivery/internal/dto/v1_0/util/DDMFormValuesUtil.java

```diff
@@ -79,6 +84,46 @@ public static DDMFormValues toDDMFormValues(
 		};
 	}
 
+	public static String toText(
```

Comment:
toText seems to abstract, could it be toContent or something like that?

Reply:
I agree. Or maybe serializeContent

Reply:
I renamed it to `serializeContent`